<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefereeInfoWidget</class>
 <widget class="QWidget" name="RefereeInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Referee Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="divisionLabel">
     <property name="text">
      <string>Division: A</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="teamYellow">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="yellowCardGroupYellow">
         <property name="title">
          <string>Yellow Cards</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="yellowCardsCounterYellow">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="cardTextLabelYellow">
              <property name="text">
               <string>Current Yellow Cards:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="numberOfCardsYellow">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="timeLeftOnYellowCardYellow">
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="format">
             <string>Card expires in: %vs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="redCardTextLabelYellow">
           <property name="text">
            <string>Red Cards:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="redCardCountYellow">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="foulLabelTextYellow">
           <property name="text">
            <string>Fouls:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="foulCounterYellow">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="keeperIdDisplayYellow">
         <item>
          <widget class="QLabel" name="keeperTextLabelYellow">
           <property name="text">
            <string>Goalie:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="keeperIdYellow">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="teamBlue">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="yellowCardGroupBlue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Yellow Cards</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="yellowCardsCounterBlue">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="cardTextLabelBlue">
              <property name="text">
               <string>Current Yellow Cards:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="numberOfCardsBlue">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="timeLeftOnYellowCardBlue">
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="format">
             <string>Card expires in: %vs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="redCardTextLabelBlue">
           <property name="text">
            <string>Red Cards:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="redCardCountBlue">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="foulLabelTextBlue">
           <property name="text">
            <string>Fouls:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="foulCounterBlue">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="keeperIdDisplayBlue">
         <item>
          <widget class="QLabel" name="keeperTextLabelBlue">
           <property name="text">
            <string>Goalie:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="keeperIdBlue">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
